charts/ratify/templates/configmap.yaml

```diff
@@ -41,7 +41,7 @@ data:
             }
         }
       },
-      "verifiers": {
+      "verifier": {
```

Comment:
so that is true but the real issue was that people were getting confused why `stores` `policies` and `verifiers` section didn't take a list of configs at that level. Instead we require they specified as a list of `plugins`. I changed them to singular because each section is just a header. So that's why anything verifier related is in the `verifier` section. Is it too confusing?